package day39_b_exception.learn;

public class FourthTry {
}
